src/UniversalRewardsDistributor.sol

```diff
@@ -98,7 +86,7 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
     /// @param distributionId The distributionId of the merkle tree distribution.
     /// @dev This function can only be called after the timelock has expired.
     /// @dev Anyone can call this function.
-    function acceptRootUpdate(uint256 distributionId) external notFrozen(distributionId) {
+    function acceptRootUpdate(uint256 distributionId) external {
         PendingRoot memory pendingRoot = pendingRootOf[distributionId];
```

Comment:
should we check that root is not 0?

Reply:
There is no reason to prevent a root updater to submit the root 0 in the specs (for example, a strategy that is revoking claims after a given time, like Safe)